CONTRIBUTING.md

```diff
@@ -44,6 +44,7 @@ development of the library, from your local clone of the fork, run
 ```bash
 (myenv) pip install -e .
 (myenv) pip install -r requirements.txt
```

Comment:
This is redundant after `pip install -e .` and should be removed.

Reply:
Yes, I thought so as well. But in a previous discussion, you had asked me to add it in or maybe I misread that comment !